### DISCUSSION TITLE
is posible use reitit for routing  in scittle in some way?

### DISCUSSION THREAD

**Discussion by mikelemus27** [borkdude/scittle#64] [Ideas]
it is posible to  use reitit for routing/navigation  in scittle in some way?
i tried to require it  and get this error:   Could not find namespace reitit.core
i just couldnt  find a way to do that   
thanks

**Comment by borkdude** [comment]
I guess reitit could be a nice addition to the suite of reagent + re-frame... right? Do you have a small example of how to use it in CLJS (and how it would be used in scittle)?

**Comment by borkdude** [comment]
One thing that concerns me about adding reitit is that it seems to be a full suite of tools, connected to coercion, spec, malli, etc etc. So adding only reitit core might lead to adding a host of new things.

Another option is to just use a JS library in a script tag and use that as a router, maybe?
